### DISCUSSION TITLE
معرفی مسیرهای یادگیری در پردیس جهانی

### DISCUSSION THREAD

**Discussion by appleusa** [community/community#64806] [Repositories]
### Select Topic Area

General

**Comment by queenofcorgis** [comment]
Hi @appleusa , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
